core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -361,6 +361,11 @@ private CarbonCommonConstants() {
 
   public static final String CARBON_INDEX_SCHEMA_STORAGE_DATABASE = "DATABASE";
 
+  @CarbonProperty
+  public static final String CARBON_INDEXFILES_DELETETIME = "carbon.index.delete.time";
```

Comment:
I feel instead of CARBON_INDEXFILES_DELETETIME, can use existing max.query.execution.time.

Reply:
agree with @ajantha-bhat , we can reuse the query execution timeout and delete as its also one hour.

Reply:
Done. Removed `CARBON_INDEXFILES_DELETETIME` and modified to use `max.query.execution.time`